.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
Is the enumeration just a set of examples or fixed steps how to reproduce? If it is the set, it should not be numbered IMO.  Also I do not understand See error step.

Reply:
It is just example. The last step means that once you make steps 1-3 then you can see the error.
BTW all of this is just template provided to me by GitHub.